v2g-liberty/rootfs/root/appdaemon/apps/v2g-liberty/modbus_evse_client.py

```diff
@@ -274,20 +273,22 @@ async def initialize(self):
 
     async def initialise_charger(self, v2g_args=None):
         self.log(f"Configuring Modbus EVSE client at {c.CHARGER_HOST_URL}:{c.CHARGER_PORT}, reason: {v2g_args}")
+
+        # Remove old client if needed.
+        if self.client is not None:
+            if self.client.connected:
+                self.client.close()
+            self.client = None
```

Comment:
Setting the client to `None` has little added value. In the next line you are assigning it a new value. I think this line can be safely removed.

Reply:
Will do